index.html

```diff
@@ -107,8 +107,25 @@ <h3 class="work-title">work 1</h3>
         </section>
         <section class="testimonial">
              <h1 class="testimonial-title underline-center">Take Their Words</h1>
-             <div class="testimonials-container" id="testimonialCard">
-               
+             <div class="testimonials-container">
+                <div class="testimonial-card">
+                    <div class="testimonial-card-img">
+                    </div>
+                    <div class="testimonial-card-content">
+                         <h2 class="testimonial-name">haben mehari</h2>
+                         <p class="testimonial-task">desk move</p>
+                         <p class="testimonial-comment">Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto deleniti sint a inventore perspiciatis? Incidunt, fuga! Aspernatur ullam consectetur aliquid deserunt alias, adipisci natus sapiente quam vero eaque! Eveniet, porro.</p>
```

Comment:
> * [ ]  Kindly follow the design guidelines for speakers section
> * Your Design
> 
> <img alt="Screenshot 2023-05-31 at 05 25 16" width="1334" src="https://user-images.githubusercontent.com/81444338/242273713-2121f5a3-7fc5-4be4-ac59-0c1f7b553d81.png">
> * Cydin Design
> 
> <img alt="Screenshot 2023-05-31 at 05 26 43" width="1352" src="https://user-images.githubusercontent.com/81444338/242273938-f8385f8a-9d1e-4bd4-8e40-01c7c459296c.png">

Kindly ensure that your speakers' section follows the design guideline. Please, note the following:

- The padding of the container
- The black check background behind the images
- You need to have at least 6 speakers so as to reflect the design
- Use actual text instead of lorem